README.md

```diff
@@ -27,6 +27,9 @@ Synopsis
     # Start current version
     pgenv start
 
+    # Show server status
+    pgenv status
```

Comment:
Can we add a paragraph with the description and an example of the status command, telling the user that the `status` is using `pg_ctl` behind the scenes?

Reply:
Sure